content/advent-2018/go-arrow.md

```diff
@@ -0,0 +1,531 @@
+
+Most of the science-y things one might carry out with the Python scientific stack (NumPy/SciPy) can be performed with [Gonum](https://gonum.org):
+
+- linear algebra, matrices,
+- differentiation,
+- statistics, probability distributions,
+- integration, optimization,
+- network creation,
+- Fourier transformations,
+- plots.
+
+There are also a few packages that enable file-level interoperability with many useful data science oriented or scientific "file formats":
+
+- [CSV](https://godoc.org/encoding/csv)
+- [SQL](https://godoc.org/database/sql)
+- [Numpy data files](https://godoc.org/github.com/sbinet/npyio)
+- [HDF5](https://godoc.org/gonum.org/v1/hdf5)
+- [FITS](https://godoc.org/github.com/astrogo/fitsio)
+- [HDFS](https://godoc.org/github.com/colinmarc/hdfs)
+- [Parquet](https://godoc.org/github.com/xitongsys/parquet-go/parquet)
+- [ORC](https://godoc.org/github.com/scritchley/orc)
+- [Cassandra](https://godoc.org/github.com/gocql/gocql)
+- ...
+
+Many basic ingredients are already there, even if some are still missing.
+
+But even if basic file-level interoperability is (somewhat) achieved, one still needs to implement readers, writers and converters from one file format to any another when integrating with a new analysis pipeline:
+
+![file formats](/postimages/advent-2018/go-arrow/arrow-copy.png)
+
+In many scenarios, this implies a large fraction of the computation is wasted on serialization and deserialization, reimplementing over and over the same features for converting from `format-1` to `format-n` or from `format-1` to `format-m`, _etc..._
+
+What if we had a common data layer between all these analysis pipelines, a common tool that would also be efficient?
+
+## Arrow
+
+What is [Arrow](https://arrow.apache.org)? From the website:
+
+```
+Apache Arrow is a cross-language development platform for in-memory data.
+
+It specifies a standardized language-independent columnar memory format for flat
+and hierarchical data, organized for efficient analytic operations on modern
+hardware.
+It also provides computational libraries and zero-copy streaming messaging and
+interprocess communication.
+```
+
+![arrow shared](/postimages/advent-2018/go-arrow/arrow-shared.png)
+
+The idea behind Arrow is to describe and implement a cross-language mechanism for efficiently sharing data across languages and processes.
+This is embodied as an `arrow::array` in C++, the reference implementation.
+Languages currently supported include C, C++, Java, JavaScript, Python, and Ruby.
+
+... and -- of course -- Go :)
+
+[InfluxData](https://www.influxdata.com/) contributed the original Go code for Apache Arrow, as announced [here](https://www.influxdata.com/blog/influxdata-apache-arrow-go-implementation/).
+Initially, the Go Arrow package implemented by [Stuart Carnie](https://twitter.com/stuartcarnie) had support for:
```

Comment:
There were some other contributors, but you're right that Stuart was the main one. Might be better not to suggest he was the only one